Automattic/wp-calypso
# Add Google My Business Stats Tab

## Specs

Adds an additional tab to the stats page with Google My Business Stats

![image](https://user-images.githubusercontent.com/2810519/38431724-03dd4ed8-397a-11e8-963b-f95a52b64e55.png)

## Testing

1. Load this PR in a local development branch
2. Navigate to the stats page any site
3. Verify that there is a "Google My Business" Tab
<img width="405" alt="screen shot 2018-04-06 at 4 32 39 pm" src="https://user-images.githubusercontent.com/2810519/38448292-2bb1cc02-39b8-11e8-96c0-2e319911ed47.png">
4. Navigate to the Google My Business Tab
5. Verify that the page looks like this (the numbers will very slightly because in an act of developer laziness a generate them randomly on page load instead of making some up )
<img width="717" alt="screen shot 2018-04-06 at 4 34 01 pm" src="https://user-images.githubusercontent.com/2810519/38448327-5be7eaaa-39b8-11e8-827e-1008ef839341.png">
